src/main/java/filter/expression/Qualifier.java

```diff
@@ -28,10 +28,10 @@
     public static final Qualifier FALSE = new Qualifier("false", "");
 
     public static final List<String> KEYWORDS = Collections.unmodifiableList(Arrays.asList(
-        "assignee", "author", "body", "closed", "comments", "created", "creator",
+        "assignee", "as", "author", "au", "body", "closed", "comments", "created", "creator",
         "date", "desc", "description", "false", "has", "id", "in", "involves",
-        "is", "issue", "keyword", "label", "labels", "merged", "milestone", "milestones",
-        "no", "nonSelfUpdate", "open", "pr", "pullrequest", "read", REPO, SORT, "state", "status",
+        "is", "issue", "keyword", "label", "labels", "merged", "milestone", "m", "milestones",
+        "no", "nonSelfUpdate", "open", "pr", "pullrequest", "read", REPO, SORT, "state", "status", "s",
         "title", "type", "unmerged", "unread", UPDATED, "user"
     ));
```

Comment:
Each string literal appears twice in this class. We can remove that duplication. e.g.
1. define an enumeration for all key words e.g. `KEYWORDS={ASSIGNEE, AUTHOR, ...}`
2. create a hasmap where the key is the string literal and the value is the enumeration value
3. during parsing, retrieve the enumeration value using the string literal and do the `switch` based on the enum value.
This can be done as a separate issue. Check with @dariusf too.